packages/core/package.json

```diff
@@ -99,4 +99,4 @@
   "nyc": {
     "extends": "../../configs/nyc.json"
   }
-}
+}
```

Comment:
@c16a can you revert this change, the newline should be present at the end of files.

Reply:
Oh sorry I was editing this in Gitpod and it looks like the new line was automatically removed. I'm trying to add one again, but it vanishes when I try to save it.

I'll try pushing from local.